CQUI.modinfo

```diff
@@ -577,23 +571,23 @@
         <ReplaceUIScript id="CQUI_TechTree">
             <Properties>
                 <LuaContext>TechTree</LuaContext>
-                <LuaReplace>Assets/UI/Screens/techtree_CQUI_basegame.lua</LuaReplace>
+                <LuaReplace>Assets/UI/Screens/TechTree_CQUI.lua</LuaReplace>
             </Properties>
         </ReplaceUIScript>
 
         <ReplaceUIScript id="CQUI_TechTree_EXP1" criteria="Expansion1">
             <Properties>
                 <LoadOrder>100</LoadOrder>
                 <LuaContext>TechTree</LuaContext>
-                <LuaReplace>Assets/Expansion1/Replacements/techtree_CQUI_expansion1.lua</LuaReplace>
+                <LuaReplace>Assets/UI/Screens/TechTree_CQUI.lua</LuaReplace>
             </Properties>
         </ReplaceUIScript>
 
         <ReplaceUIScript id="CQUI_TechTree_EXP2" criteria="Expansion2">
             <Properties>
                 <LoadOrder>200</LoadOrder>
                 <LuaContext>TechTree</LuaContext>
-                <LuaReplace>Assets/Expansion2/Replacements/techtree_CQUI_expansion2.lua</LuaReplace>
+                <LuaReplace>Assets/UI/Screens/TechTree_CQUI.lua</LuaReplace>
```

Comment:
if the game is expansion2, does this result in the file being loaded twice (for the loadorder 0 case above and here with loadorder 200)?    I thought since that first instance has no criteria attached it loads...

Reply:
No. The file is IMPORTED only once because there is only 1 entry in ImportFiles. Then after all ReplaceUIScript commands are performed, it is determined which file is an actual .lua for a context. Then this final file is EXECUTED once.
Specific ReplaceUIScript are needed because there is a replacement for XP1 and XP2 in the vanillla game. Better safe than sorry because as long as no LoadOrder is used, then the actual load order of such components is undetermined. That is all XP2 files should have its own entry point.

Reply:
Good to know, thanks!